### ISSUE TITLE
[FR] Better fuzzy search

### ISSUE THREAD

**Issue by marcjulianschwarz** [marcjulianschwarz/obsidian-raycast#34]
See https://github.com/raycast/extensions/issues/3030 for more discussions on this topic.

**Comment by chriswep** [comment]
Is it expected behaviour that the plugin currently only searches the full query string as it is? If i enter multiple words it only finds them if they appear exactly in this order in a note.

Thanks for providing this plugin!

**Comment by KevinBatdorf** [comment]
Yeah, but I'm going to look at search this week/weekend and see if there's a good way to improve both search and also handling large amounts of data
